Switch out col-md class for col-sm for consistency

clarissastark opened the pull request:
I rule and have completed some work on Case Manager that's ready for review!

This pull request makes the following changes:
* Standardize bootstrap col attributes and changing the app/views html.erb files to standardize use the "col-sm" class instead of "col-md"

It relates to the following issue #s: 
* Fixes  #144

colinxfleming commented:
Thanks a ton for getting this in, @clarissastark ! And welcome aboard, we're glad to have you!

I'm going to use this PR to train @ajohnson052 on reviewing, so expect feedback next week.

Thanks much!

clarissastark commented:
Great, thanks Colin! Very glad to be here :)

ajohnson052 commented on app/views/pregnancies/_call_log.html.erb:
```diff
@@ -1,5 +1,5 @@
 <section id="call_log" class="call-info tab-pane">
-	<div class="margin-bottom col-md-12">
+	<div class="margin-bottom col-sm-12">
 		<h1 class="border-bottom title">Call Log</h1>
```
There's a merge conflict here as well because we're using an h2 for 'Call Log' now

ajohnson052 commented:
Thanks so much for making this change! It looks like some merge conflicts cropped up because of updates near your changes. Can you go ahead and resolve those? You'll want to keep the updated ids and titles while keeping your updates to include the 'col-sm-12' class. I tagged a couple of them to explain further, and if anything gets confusing just let me know. I'd be happy to help out!

clarissastark commented:
Sure thing, I addressed those two issues – I also pulled down from the master and addressed a couple of other merge conflicts in the views/pregnancies folder, and pushed changes up.

Please just let me know if there are issues!

ajohnson052 commented:
Hey Clarissa, so sorry but I have to ask you to pull in the most recent master one more time. Right now some tests on your branch are failing, but it's not from anything you've done! The tests have just been updated since the last time you merged in code. Also, I don't think I was being very clear on the titles. Can we keep <h2>Notes</h2> in app/views/pregnancies/_notes.html.erb and <h2>Call Log</h2> in app/views/pregnancies/_call_log.html.erb? This will keep things consistent with the other sections. Thanks!

colinxfleming commented:
@clarissastark I'm going to pick this up and get it merged, so we can make sure your hustle's in for Thursday. Thanks much, and thanks @ajohnson052 for reviewing and pushing this along!
